libenv/sysinfo.c

```diff
@@ -1138,13 +1138,25 @@ static void OSReleaseParse(EvalContext *ctx, const char *file_path)
         char *os_release_id = SafeStringDuplicate(const_os_release_id);
         char *os_release_version_id = SafeStringDuplicate(const_os_release_version_id);
 
+        const char *alias = NULL;
```

Comment:
Could be defined inside the `if (os_release_id != NULL)` below, right?

Reply:
Thanks, fixed.